e2e/testcases/reconciler_manager_test.go

```diff
@@ -1116,3 +1120,91 @@ func filterResourceMap(resourceMap map[string]v1beta1.ContainerResourcesSpec, co
 	}
 	return filteredMap
 }
+
+// TestReconcilerManagerRSyncCRDMissing validates that the reconciler-manager
+// stays up when the RootSync CRD is deleted.
+func TestReconcilerManagerRootSyncCRDMissing(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.ACMController,
+		ntopts.WithDelegatedControl, ntopts.Unstructured)
+
+	reconcilerManagerKey := client.ObjectKey{
+		Name:      reconcilermanager.ManagerName,
+		Namespace: configsync.ControllerNamespace,
+	}
+
+	t.Log("Validate the reconciler-manager deployment is healthy")
+	nt.Must(nt.Validate(reconcilerManagerKey.Name, reconcilerManagerKey.Namespace, &appsv1.Deployment{},
+		testpredicates.StatusEquals(nt.Scheme, kstatus.CurrentStatus)))
+
+	nt.Must(nomostest.UninstallRootSyncCRD(nt))
```

Comment:
This should also register a cleanup to call `InstallRootSyncCRD` in case it errors before reaching the end of the test

Reply:
i think test setup installed configsync from scratch every time

Reply:
That happens at the beginning and end of the test suite, but for shared test environments it doesn't get installed between each test case. So if this test failed then the RootSync CRD may be missing for subsequent tests which use the same cluster